Python/Product/ExternalProfilerDriver/DWJsonModel.cs

```diff
@@ -0,0 +1,179 @@
+// Python Tools for Visual Studio
+// Copyright(c) 2018 Intel Corporation.  All rights reserved.
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver {
+
+    // Most numerical integer data is represented in dwjson as a 64-bit unsigned integer
+    // Due to JSON limitations, this is represented as a 2D named tuple, where the fields are
+    // "l" (low) and "h" (high), each containing a 32-bit *signed* integer.
+    // In this way, the max value (0xffffffffffffffff) is represented as {"h" : -1, "l" : -1}
+    public class LongInt {
```

Comment:
@huguesv , I've started work on string localization in 105bb6e79, please take a look?  In particular, I'm not sure if the use of string interpolation is appropriate, please let me know.

Reply:
Localized all strings I could identify, work in ffccbe713

Reply:
Strings should not be concatenated in code, because the concatenation order may differ based on language.
We usually have the localizable string contain the format parameter.
```
<data name="ProcessRunnerInferiorOutputPrefix" xml:space="preserve">
  <value>From process: {0}</value>
</data>
```
Then:
```
Console.WriteLine(string.Format(CultureInfo.CurrentCulture, Strings.ProcessRunnerInferiorOutputPrefix, e.Message));
```

`CultureInfo.CurrentCulture` is needed for the call to `Format`, otherwise we'll have static analysis errors.

Reply:
BTW I know this is a pain, I had to extract all the strings from code for the whole product over a year ago...

Reply:
Oh, I don't question the value (and begrudge the time) it's just that I really *really* would like to see this merged this week. :)  I thank you for the effort of going through the strings in the rest of the codebase, by the way, *that* must've been quite a chore.

Reply:
Replaced all string interpolation with string.format as recommended in 13a67afc0